### ISSUE TITLE
Implement feature to have catwalks connect to walls.

### ISSUE THREAD

**Issue by ZombiePwner1** [BEEmod/BEE2-items#1472]
Like this model here hanging_walkway_128b
![20170123181114_1](https://cloud.githubusercontent.com/assets/18079316/22199783/58136cf8-e197-11e6-98a9-ff3f476e5110.jpg)

**Comment by ZombiePwner1** [comment]
the hanging_walkway_128b should appear when there is a long catwalk and it connects to a wall to support it

**Comment by TeamSpen210** [comment]
This definitely should be done, after I redo how supports work to attach to known surfaces.

**Comment by ZombiePwner1** [comment]
Yes but it should only appear when there are walls next to the catwalk and it's abit long if there is no walls there should be support trusses underneath the catwalk.
